Assignment 2/Part 6 - Fortification/FortificationPhase.cpp

```diff
@@ -92,5 +92,36 @@ void FortificationPhase::fortify(int player, Map& m ) {
         }
 
     }
-    
+
+	string message = "PHASE_OBSERVER|Moved " + to_string(numberOfTroops) + " troops from " 
```

Comment:
I think this class is no longer used due to our new implementation of PlayerStrategyPatterns.
You can add this although I fear it may not be used.